### ISSUE TITLE
10.6 is broken and destroys ability to downgrade

### ISSUE THREAD

**Issue by bengalih** [jellyfin/jellyfin#3707]
- OS: Windows
 - Jellyfin Version: 10.6

Upgraded to 10.6 and had multiple issues including https://github.com/jellyfin/jellyfin/issues/3705

Tried to backrev to 10.5.5 and apparently the upgrade migrated the configs, especially users so that it is impossible to downgrade again.  So now stuck with a broken 10.6 and no way to go back.

Yes, I should have backed up my program data before, but JF should also have notified it was going to upgrade the DB and going back was impossible.  I read the info here:
https://jellyfin.org/posts/jellyfin-10-6-0/

And nothing there indicates I would have this issue.

**Comment by dkanada** [comment]
Not sure why this is labeled bug, because there's no bug report here. Also, we always request that people read the release notes when upgrading, and avoid using automated systems to prevent issues during upgrades. You can find the [release notes](https://github.com/jellyfin/jellyfin/releases/tag/v10.6.0) on GitHub, which go over every possible issue for each release.

**Comment by bengalih** [comment]
Guilty of not reading the release notes, but surprised this wasn't highlighted smack dab in the middle of the blog.
As plugins are now broken and apparently reverse proxy as well, can you provide any more info as per the release notes of how we can edit to back rev?

I'm ok with loosing my users (and I'm not sure what "Activity database" refers to specifically, but might be ok with losing that).  I'd just like to get back to a workable version without losing all my libraries until these issues are worked out.  I was having issues with Notifications and DLNA not working so was hoping to do a quick update to 10.6.0 before I filed bugs against those, but now I'm stuck with nothing workable.

thanks.

**Comment by dkanada** [comment]
I'm actually not sure about the downgrade process myself but I can tell you that reverse proxy is most definitely not broken for most users, because nearly the entire development team uses a myriad of solutions for that.

**Comment by bengalih** [comment]
> I'm actually not sure about the downgrade process myself but I can tell you that reverse proxy is most definitely not broken for most users, because nearly the entire development team uses a myriad of solutions for that.

Perhaps not all reverse proxy, but as per my comments here:
https://github.com/jellyfin/jellyfin/issues/3705#issuecomment-663940636

It appears that internally JF is trying to use http URIs which won't work when the site is accessible only over https.  I'm not sure that's the reason the plugins is broken for me, but I can't dispute the logs trying to access an http site that it will never reach.

**Comment by bengalih** [comment]
Closing as reactionary  : ) ... but My reverse proxy Chromecast configuration is totally broken.  I will open a new ticket to discuss and gather you guys the appropriate info.

**Comment by PrplHaz4** [comment]
> 
> 
> Closing as reactionary : ) ... but My reverse proxy Chromecast configuration is totally broken. I will open a new ticket to discuss and gather you guys the appropriate info.

Here's some notes on downgrading - not sure if they work...etc but might get you out of a jam if it's still relevant.
https://github.com/jellyfin/jellyfin-docs/pull/372/files

I wonder if the reason this hasn't come up before is that most people may be redirecting 80 to 443 (I know I do), which may hide the issue?

**Comment by bengalih** [comment]
> > Closing as reactionary : ) ... but My reverse proxy Chromecast configuration is totally broken. I will open a new ticket to discuss and gather you guys the appropriate info.
> 
> Here's some notes on downgrading - not sure if they work...etc but might get you out of a jam if it's still relevant.
> https://github.com/jellyfin/jellyfin-docs/pull/372/files
> 
> I wonder if the reason this hasn't come up before is that most people may be redirecting 80 to 443 (I know I do), which may hide the issue?

Thanks.  I found a recent enough backup that I was able to restore for settings/users.  Some of my metadata probably had to be redownloaded.

I'm 90% sure that it is exactly right, that JF expects 80 to be available and for the reverse proxy to automatically do the redirection.  I know there are *many* home users whose ISPs block incoming port 80 and I don't think that requiring 80 to be available (even if for just a redirect) should be required for a modern web app that can/should be fully secured over 443.

I'm going to look about playing around with some internal redirects and opening port 80, but I don't have high hopes on this.  I just hope someone acknowledges that this is a real issue for a configuration which worked perfectly fine under 10.5.5, and is now non-funcitonal.
